core/controllers/editor_test.py

```diff
@@ -67,13 +67,15 @@ def setUp(self):
 
     def assert_can_edit(self, response_body):
         """Returns True if the response body indicates that the exploration is
-        editable."""
+        editable.
+        """
```

Comment:
Is the "triple quote" moving to the next line the desired result?

Reply:
This is fine :)